x-pack/test/send_search_to_background_integration/tests/apps/management/search_sessions/sessions_management.ts

```diff
@@ -18,20 +19,24 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   ]);
   const searchSessions = getService('searchSessions');
   const esArchiver = getService('esArchiver');
-  const retry = getService('retry');
+  const log = getService('log');
 
-  // FLAKY: https://github.com/elastic/kibana/issues/89069
-  describe.skip('Search search sessions Management UI', () => {
+  describe('Search sessions Management UI', () => {
     describe('New search sessions', () => {
       before(async () => {
         await PageObjects.common.navigateToApp('dashboard');
+        log.debug('wait for dashboard landing page');
+        retry.tryForTime(10000, async () => {
```

Comment:
This is the main reason why the test was flaky: using `dashboard.loadSavedDashboard` has flaky behavior unless we make sure we first on the dashboard landing page and not a loading screen.

Reply:
doesn't `navigateToApp('dashboard');` always take us to the landing page?  I would think if that navigate method always sets the URL and uses the timestamp hash that it would reliably be on the landing page?  I think some dashboard tests did something to avoid the timestamp hash in the URL to preserve state.  Maybe that's related.

Reply:
It takes us to the landing page, which could actually be displaying a loading screen. This retry waits until any loading screen is cleared.